.github/workflows/android-ui-test.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: '11'
+      - name: create google_services.json
```

Comment:
Should we split this out into a separate script since it's nothing to do with ui tests?

Reply:
Do you mean running it as a separate workflow?

Reply:
Yes, just because this one is named android-ui-test.  Maybe it just needs to be renamed?

Reply:
I think we need to run this step for every workflow that builds the android app, because the file won't be persisted across jobs

Reply:
Sure, I was thinking we could break things like this out into a reusable workflow so we don't need to duplicate the logic and we can keep the main workflows clean 
https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
Ahhh I see what you mean. Yeah that's a good idea!
I'll create a reusable workflow once I get it working